### DISCUSSION TITLE
How to share a package devloped under a normal user to system wide users

### DISCUSSION THREAD

**Discussion by Weicai-Ling** [cockpit-project/cockpit#20432] [Q&A] [answered]
Hello experts: 
I install cockpit by dnf install cockpit on Rocky Linux 9.3 in root and downloaded starter-kit package in a normal user. The package can be seen with the normal user login the web UI after running `"ln -s /home/user/startkit/dist ~/.local/share/cockpit/starter-kit"`. I want the package to be seen by other users, therefore I run the command of `"ln -s /home/user/startkit/dist /usr/local/share/cockpit/starter-kit"` in root, the normal user and other users cannot seen the package after login the web. 
Who could please help to advise if the package developed in normal user been seen by system wide users ? 

THANKS ! 
Forrest Weicai Ling

**Comment by martinpitt** [comment, marked as answer]
In most operating systems, only the owner of a home directory can access it, they are usually not readable by any other users. So so do that, please either do `sudo make install` or *copy* the installation into /usr/local:
```
sudo cp -r /home/user/startkit/dist /usr/local/share/cockpit/starter-kit
```

**Comment by Weicai-Ling** [comment]
@martinpitt THANKS for your quick reply and help. It WORKS !!!

Really Appreciate again for your reply !

Forrest Weicai Ling

**Reply by martinpitt** [reply]
Nice!
